src/QuartzNET-DynamoDB/JobStore.cs

```diff
@@ -832,7 +846,39 @@ public void ResumeTrigger(TriggerKey triggerKey)
 
         public IList<string> ResumeTriggers(GroupMatcher<TriggerKey> matcher)
         {
```

Comment:
Possible missing Lock statement. Would it matter if two threads executed this concurrently? Or if this operation executed concurrently with another, say PauseTriggers? Possibly not... I don't even know if the scheduler calls this or if it exists only for UI's...